source/includes/steps-atlas-cli-deploy-local-prereqs.rst

```diff
@@ -0,0 +1,56 @@
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the dependencies.
+
+      a. Install the :ref:`{+atlas-cli+} <install-atlas-cli>`.
+
+         If you use `Homebrew <https://brew.sh/#install>`__, you can
+         run the following command in your terminal:
+
+         .. code-block::
+
+            brew install mongodb-atlas-cli
+               
+         For install instructions on other operating systems,
+         see :ref:`install-atlas-cli`.
+
+      #. Install `Docker <https://www.docker.com/>`__.
+
+         - For MacOS or Windows, install `Docker Desktop v4.31+ <https://docs.docker.com/desktop/release-notes/#4310>`__. 
+         - For Linux, install `Docker Engine v27.0+ <https://docs.docker.com/engine/release-notes/27.0/>`__.
+
+         .. note::
+
+            Docker requires a network connection for pulling and caching 
+            MongoDB images.
+
+            `Podman v5.0+ <https://podman.io>`__ is also supported for Linux RHEL versions.  
```

Comment:
For RHEL, you can also use |podman|.

Suggestion: Add Podman, Docker and Homebrew to the snooty.toml replacements. If you do, replace them in this file with |homebrew|, |docker|, and |podman|.